app/components/trainees/assessment_details/view.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= render_component SummaryCard::View.new(
+  title: "Assessment details",
+  heading_level: 2,
+  rows: [
+    {
```

Comment:
This is more of a 'way of working' question, but as we're not implementing the "Have they met this standards for QTS?" page in this sprint and only recording the date, do we want to remove this for now?

Reply:
agreed and doen https://github.com/DFE-Digital/register-trainee-teachers/pull/235/commits/9672f12c607a2789fff0ea03eabc16d87a320b7d